frontend/mr-admin-flate/src/pages/tiltakstyper/DetaljerTiltakstypePage.tsx

```diff
@@ -26,10 +26,7 @@ export function DetaljerTiltakstypePage() {
   if (!optionalTiltakstype.data) {
     return (
       <Alert variant="warning">
-        Klarte ikke finne tiltakstype
-        <div>
-          <Link to="/">Til forside</Link>
-        </div>
+        Klarte ikke finne tiltakstype <Link to="/">Til forside</Link>
```

Comment:
Nå kommer vel alt på en linje? `Klarte ikke finne tiltakstype <link kommer her>`?

Reply:
Skulle det komme på to linjer? Det kom på en linje men uten mellomrom mellom tekst og link